Examples/Website/Server.py

```diff
@@ -0,0 +1,45 @@
+# importing flask framework to make a server
```

Comment:
This is another great example, and is well done, but unfortunately, it is against sports-reference.com's terms of service to host information directly from their site without changes on a new website, so we can't include this. Granted, you _could_ do some analysis on the numbers, such as use a machine learning based exclusively on the data pulled from sports-reference.com to predict some new outcome or stat, and create a website with that data. But, as-is, we will need to take this down to not violate their terms of service.